### ISSUE TITLE
problems about pose optimization and steep slopes

### ISSUE THREAD

**Issue by zjtde1990** [DRosemei/RoMe#18]
Thank you for your great result. But I have some problems, can you give me some advice?
1. For the problem of inaccurate input poses, can I optimize the camera's interframe poses during the optimization process? How can I do that? Can you give me some advice?
2. What is the optimizable range of height and is it related to the learning rate of z-mlp? What should be done about steep slopes? 
3. The rendering image is fuzzy, how to improve the rendering effect of the view? I tried to modify the loss fuction to ssim+l1, but I didn't get good results. Hope to get some suggestions. 
Thank you so much~

**Comment by DRosemei** [comment]
@zjtde1990 
1. You can refer [here](https://github.com/DRosemei/RoMe/blob/main/models/pose_model.py#L29). While in our experiments, it is hard to refine source poses because of weak supervision. Accurate poses will get better results.
2. Optimizable range depends on learning rate of z-mlp and it is no need to give larger learning rate because road surface is mostly smooth. You can refer to https://longtimenohack.com/streetsurf_web to pretrain your mlp.
![image](https://github.com/DRosemei/RoMe/assets/27892562/79130432-cf8e-4b9b-9026-3a7afd54340c)
3. You can decrease bev_resolution but it will use larger GPU memory.

**Comment by zjtde1990** [comment]
@DRosemei 
Thank you for your reply.
In addition, what are your suggestions for the loss function? For example, L1, L2 ,ssim , MS-ssim+L1, etc., which is the better?

**Comment by DRosemei** [comment]
@zjtde1990 Hi. These functions may not be useful as supervision is so weak. You can try to use sfm/mvs/lidar points for strong supervison and you can refer to our new work https://arxiv.org/abs/2309.11754

**Comment by zjtde1990** [comment]
@DRosemei Ok, I will read it in detail. Thank you very much~
